doc/spec.adoc

```diff
@@ -231,10 +231,7 @@ A blocklist applies only to the channel that defined it, not its required channe
 
 #### Resolving artifact version
 
-During artifact version resolution, a stream matching artifact GA is located in the channel. If the stream uses concrete versions, that version of the artifact is resolved and returned to the user.
```

Comment:
I'm not sure if we should be changing this. IMO blocklists don't make much sense when a channel is using a concrete version.

The idea of a blocklist was to block one version of an artifact, letting the channel fallback on an earlier version of that artifact. IMO when a single version is provided by a channel manifest, blocklisting it breaks that channel. 

@jmesnil what do you think should be the expected behaviour?

Reply:
I'm not sure what the implementation should be but the existence of a blocklist means that you don't want that specific version(s). It's not a bad thing to break the channel. This will indicate an invalid configuration.

Reply:
As a note, another use case that came up recently is a situation where artifacts are resolved from multiple channels. In that scenario the artifact could have multiple versions and blocking one of those might make sense